lua/bufferline/render.lua

```diff
@@ -113,7 +116,7 @@ local function render()
     local mod = is_modified and 'Mod' or ''
 
     local separatorPrefix = hl('Buffer' .. status .. 'Sign')
-    local separator = status == 'Inactive' and
+    local separator = is_inactive and
```

Comment:
Boolean comparison is probably faster than running a bunch of string comparisons, and `is_inactive` was the only `is_<status>` missing. 

I'll take it out if you want me to.